added good to know in `/app` based pages and layout in nested layouts (#51558)
vercel/next.js: docs/02-app/01-building-your-application/01-routing/02-pages-and-layouts.mdx

}
```

> **Good to know**:
>

> > - Only the root layout can contain `html` and `body` tags.

Reviewer: ```suggestion
> - Only the root layout can contain `<html>` and `<body>` tags.
```

> > - Layouts defined inside a folder should not contain `html` or `body` tags.

Reviewer: ```suggestion
> - Only the root layout can contain `html` and `body` tags.
```

Is this what you mean?

Author: Yes.
Pushing the change.
